src/frontend/src/components/common/Navbar/index.tsx

```diff
@@ -20,7 +24,9 @@ function NavBar({ pages }: INavbarPages): ReactElement {
           ) : (
             <li key={page.id}>
               <NavLink
-                className="bcgovbtn bcgovbtn__secondary--dark"
+                className={`bcgovbtn bcgovbtn__tertiary bcgovbtn__tertiary--dark ${
+                  page.link === currentPath && 'active'
```

Comment:
Doing it this way will add "false" to class name when this condition is not true

```suggestion
                  page.link === currentPath ? 'active' : ''
```